src/app/app.component.ts

```diff
@@ -56,12 +56,22 @@ export class AppComponent implements OnInit, OnDestroy {
       this.apiKey = params.get('sdkKey');
       this.baseUrl = params.get('baseUrl');
       this.featureFlagKey = params.get('featureFlagKey');
+      let hideControls = params.get('hideControls');
 
-      if (!this.featureFlagKey) { this.featureFlagKey = 'isAwesomeFeatureEnabled'; }
+      if (!this.featureFlagKey) { this.featureFlagKey = ''; }
```

Comment:
Why is this better?

Reply:
- the old version's goal was to set a default feature flag, but the same functionality is implemented elsewhere too
- at the same time, this line prevented me from checking if there is a valid featureFlagKey set in the URL